CHANGELOG.md

```diff
@@ -5,6 +5,48 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.3.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.3.0) - 2023-09-25
+
+This release provides support for the Spend before Sync synchronization
+strategy, and contains the following updates:
+
+### Security
+
+- Fix database concurrency bugs that could have led to panics or incorrect history tree data (#7590)
+
+### Added
+
+- Add missing elasticsearch flag feature to lib docs (#7568)
+- Add missing Docker variables and examples (#7552)
+
+#### Spend before Sync support
+
+- Add state requests and support code for the `z_getsubtreesbyindex` RPC (#7408)
+- Implement the `z_getsubtreesbyindex` RPC (#7436)
+- Add snapshot tests for the `z_getsubtreesbyindex` RPC (#7514)
+- Test the `z_getsubtreesbyindex` RPC (#7515, #7521, #7566)
```

Comment:
Optional: could be combined

Reply:
Done in [59f7bc1](https://github.com/ZcashFoundation/zebra/pull/7610/commits/59f7bc1bdacbcd14998b250aa6f5da55fac06c81).